### ISSUE TITLE
DataMovementServices is holdover from DHF 1.0

### ISSUE THREAD

**Issue by grechaw** [marklogic-community/marklogic-data-hub#613]
I notice that this file implements some DMSDK stuff for DHF.

https://github.com/marklogic-community/marklogic-data-hub/blob/master/marklogic-data-hub/src/main/java/com/marklogic/client/datamovement/impl/DataMovementServices.java

This code calls an endpoint that does not exist.  In some classpath resolutions (gradle) this file is chosen over the one supplied by the Java Client API.

This workaround in my project solves the problem.

```
compile('com.marklogic:marklogic-data-hub:2.0.2') {
        exclude module: 'marklogic-client-api'
    }
```

In short, having this file clobber on the package structure is probably no longer necessary, and causes unexpected build issues in some systems.

MarkLogic version is 9 nightly (doesn't matter).

Here is what happens.  Reproducible in IDEA and gradle command line but not eclipse:

```
package com.marklogic.ic360.services;

import com.marklogic.client.DatabaseClient;
import com.marklogic.client.DatabaseClientFactory;
import com.marklogic.client.datamovement.DataMovementManager;
import com.marklogic.client.datamovement.WriteBatcher;
import org.tmatesoft.svn.core.SVNException;

public class TestThisStuff {


    public static void main(String[] args) throws SVNException, InterruptedException {

         DatabaseClient srcClient = DatabaseClientFactory.newClient("localhost", 8000,

        new DatabaseClientFactory.DigestAuthContext("admin", "admin"));

        DataMovementManager srcMoveMgr = srcClient.newDataMovementManager();

        WriteBatcher qb = srcMoveMgr.newWriteBatcher();

    }

}
```

Throws:

```
Exception in thread "main" com.marklogic.client.FailedRequestException: Local message: failed to read resource at resources/forest-info: Internal Server Error. Server Message: RESTAPI-INVALIDREQ: (err:FOER0000) Invalid request:  reason: Extension forest-info or a dependency does not exist: XDMP-MODNOTFOUND: (err:XQST0059) Module /marklogic.rest.resource/forest-info/assets/resource.xqy not found . See the MarkLogic server error log for further detail.
	at com.marklogic.client.impl.OkHttpServices.checkStatus(OkHttpServices.java:4317)
	at com.marklogic.client.impl.OkHttpServices.getResource(OkHttpServices.java:3039)
	at com.marklogic.client.datamovement.impl.DataMovementServices.readForestConfig(DataMovementServices.java:45)
	at com.marklogic.client.datamovement.impl.DataMovementManagerImpl.readForestConfig(DataMovementManagerImpl.java:170)
	at com.marklogic.client.datamovement.impl.DataMovementManagerImpl.getForestConfig(DataMovementManagerImpl.java:165)
	at com.marklogic.client.datamovement.impl.DataMovementManagerImpl.newWriteBatcher(DataMovementManagerImpl.java:110)
	at com.marklogic.ic360.services.TestThisStuff.main(TestThisStuff.java:21)

Process finished with exit code 1
```

**Comment by paxtonhare** [comment]
@grechaw this is the "creative fix" we used to get DMSDK running on ml8. That forest-info file should be deployed by the hub for 8 and 9 and thus you shouldn't be seeing this issue.

It looks like we will probably be dropping ML8 support and thus this quality code will be removed.

**Comment by popzip** [comment]
Related to #618

**Comment by popzip** [comment]
@aebadirad does deprecation of ml8 in 2.0.3 address this issue?

**Comment by aebadirad** [comment]
I removed this code for the 2.0.3 release, and it is currently 'fixed' in develop.

**Comment by aebadirad** [comment]
This has been released in 2.0.3.
